### PR TITLE
Some clothing yaml tweaks

### PR DESCRIPTION
**Pull request by Mith-randalf**
## About the PR 
Fixes some names & descriptions, adds armour to clothes that are clearly supposed to have it, and tweaks some values (like making the HoS trenchcoat not be worse against bullets than default sec armour, adding rad armour to CE jumpsuit as per its description) etc.

Some of it might be cOnTrAvErSiAl so let me know if any of it needs changing.

Didn't do anything to helmets, I assume this is all kinda placeholder til bodypart damage is a thing and didn't want to accidentally make broken combos.

**Screenshots**

**Changelog**

:cl:
- tweak: Fixed some clothing names, applied armour to previously unarmoured clothes, tweaked some armour values.

### CODE REVIEW

File: Resources/Prototypes/Entities/Clothing/OuterClothing/suits.yml [hunk 1]
```diff
@@ -8,6 +8,13 @@
     sprite: Clothing/OuterClothing/Suits/bombsuit.rsi
   - type: Clothing
     sprite: Clothing/OuterClothing/Suits/bombsuit.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.15 #big blast protection, a little protection from heat, negligable cushioning from other damage cos its thick
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/suits.yml:14]: 0.15 might be a liiiiittle too good, a specific 'explosion-resistance' thing might be better but out of scope obviously

File: Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml [hunk 2]
```diff
@@ -41,6 +41,10 @@
     sprite: Clothing/Uniforms/Jumpskirt/ce.rsi
   - type: Clothing
     sprite: Clothing/Uniforms/Jumpskirt/ce.rsi
+  - type: Armor
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml:44]: didnt change the normal jumpsuit or am i insane?

**Review by mirrorcult — CHANGES_REQUESTED** [review]

**Comment by mirrorcult** [comment]
changelog should be made less vague

File: Resources/Prototypes/Entities/Clothing/OuterClothing/suits.yml (hunk 1 above)
Comment (Mith-randalf) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/suits.yml:14]: How about .2?

File: Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml (hunk 2 above)
Comment (Mith-randalf) [reply Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml:44]: You're not insane, I neglected to save my change like a dumb. Fixed it now.

**Comment by Mith-randalf** [comment]
> changelog should be made less vague

updated

File: Resources/Prototypes/Entities/Clothing/OuterClothing/suits.yml (hunk 1 above)
Comment (mirrorcult) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/suits.yml]: sure

**Pull request merged**